Feat (Doc): Tari FAQ (#6996)
tari-project/tari: docs/src/Tari_FAQ.md

- [Technical Characteristics](#technical-characteristics-of-tari) - Covers the technical characteristics of Tari. Network throughput, address format, consensus model and more.
- [Development](#development) - Questions related to development and development resources.
- [Exchanges](#exchanges)- Information specific to exchanges.

## General
### What are the official channels/website/social network profiles for Tari.
- Official Website: [www.tari.com](www.tari.com)
- Twitter: [https://twitter.com/tari/](https://twitter.com/tari/)
- Discord: [https://discord.gg/tari](https://discord.gg/tari)
- Telegram: [https://t.me/tariproject](https://t.me/tariproject)

### What are the official wallets for Minotari
Users have the three options available:

- [The Minotari Console Wallet](https://tari.com/downloads/) - Part of the Tari suite of applications. Provides a console-based wallet for transactions and interactions with the blockchain

> - [Aurora](https://aurora.tari.com) - A mobile application for Android and iOS.

Reviewer: this is going to be tari universe mobile?

Author: Alright, noted. I will amend and keep that in mind going forward in the future.

- Tari Universe: A desktop-based client that handles the installation and configuration of Tari in a simple installation. Recommended for the average user.

Additional, dependent components that aren't specific to Tari are:

- XMRig: Third-party Monero mining software. Used in conjunction with the `minotari_merge_mining_proxy` to mine both Monero and Tari.
- Tor: Provides network privacy and NAT traversal. Enables secure, anonymous communication between Tari nodes.

### Are there any Tari blockchain explorers?
The mainnet explorer will be [https://explore.tari.com](https://explore.tari.com).

Please note that only Testnet explorer is currently live: https://explore-nextnet.tari.com/

## Tokenomics
### Is there any documentation available describing the tokenomics of Tari?
Users can refer to the following: [https://tari.substack.com/p/tari-tokenomics](https://tari.substack.com/p/tari-tokenomics)

### Is there an initial total supply? And is the supply fixed?

> Excluding premine: 0 at genesis, 21 billion XTM once tail emission starts. 

Reviewer: you say 0 here, few lines later you correctly state `Protocol Infrastructure and Grants: 40% (756 million tokens) available at launch for protocol liquidity, `

No. Swaps must be performed manually at this time.

### Is Tari account-based?
No. Tari uses the UTXO model, similar to Bitcoin. As such there is no specific account tied to a user's transactions on Minotari. The closest comparison is the private keys associated with the wallet, which are you used to signing of transactions and validating existing UTXOs for things like calculating balance, etc.

### How often is a block mined / what is the interval per block / what is the block production rate?
Blocks are mined every 120 seconds.

### What is the precision of Minotari?
Minotari can be traded at 0.000001 XTM. Fractional Minotari is referred to as Microtari (μT).

### What coins is Tari similar to?
Bitcoin, Grin, Beam, Litecoin

### Does Tari have "super nodes"?
No. This is sssuming that "super nodes" in this context is a high-performance, specialized node that play a critical role in validation, governance, or network infrastructure, then **no**. All base nodes have an equal role in ensuring the security of the network.

### What is the transaction expiration policy in the transaction pool? Is there a timeout duration after which unconfirmed transactions are removed?

> Completed transactions are never removed. Pending transactions have a timeout of three (3) days; if they have not been completed within that period, the base node will reject it and it will be removed from the mempool, meaning that it will not be added to the blockchain.

Reviewer: ```suggestion
Completed transactions are never removed. Pending transactions have a timeout of three (3) days; if they have not been completed within that period, the wallet willremove that transaction and mark it as canceled.
```
Pending tx never leave the wallet

Author: Noted and amended.

- Additional minor documentation can be found in: https://github.com/tari-project/tari/blob/development/docs/src/
- The node and wallet require grpc to interact with the API. The proto files include documentation: 
    - Node: https://github.com/tari-project/tari/blob/development/applications/minotari_app_grpc/proto/base_node.proto
    - Wallet: https://github.com/tari-project/tari/blob/development/applications/minotari_app_grpc/proto/wallet.proto
- There are some examples and more detailed explaination of the api here: https://github.com/tari-project/tari/blob/development/docs/src/API_GRPC_Explanation.md

You can also review the Wallet FFI here: 

### What are the rules governing the construction of addresses in Tari?
You can read more [here](https://rfc.tari.com/RFC-0155_TariAddress), but we have summarized below:

The address is a base58-encoded and adheres to the following bytes/rules:
- 0 - Network: Determines which network the address belongs to.
- 1 - Features: Indicates whether the feature set for this address is for a one-sided address or an interactive address. Valid values for these are 1 and 2 respectively.
- 2 to 33 - Public view key
- 34 to 65 - Public Spend Key

> - 66 - Checksum using the DammSumm algorithm using 0,1 and the Public Spent Key to calculate.

Reviewer: ```suggestion
- 66 - Checksum using the DammSumm algorithm 
```
Its the entire thing

Author: Noted

### Does Tari support memo functions like EOS?

> No.

Reviewer: ```suggestion
Yes, its called PaymentID, and is encrypted on each output. 
```

Author: Noted.
